clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -478,7 +482,7 @@ public void abortIncompleteBatches() {
      */
     private void abortBatches() {
         for (RecordBatch batch : incomplete.all()) {
-            Deque<RecordBatch> dq = dequeFor(batch.topicPartition);
+            Deque<RecordBatch> dq = getDeque(batch.topicPartition);
```

Comment:
I don't have a strong preference, it seemed a bit more resilient to changes elsewhere, but it is a private method as you said.